src/formatDate.js

```diff
@@ -50,7 +50,33 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const partOfDate = date.split(`${fromFormat[3]}`);
```

Comment:
Avoid using template literals for a single variable. Use 'fromFormat[3]' directly.